src/components/atoms/chef/CircleChef.tsx

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+
+interface CircleChefProps {
+  icon: string;
+  name: string;
+}
```

Comment:
このコンポーネント押下でシェフページ？に遷移しそうなので、urlも受け取れるようにしておいた方がいいかなと思いましたー！
```suggestion
interface CircleChefProps {
  icon: string;
  name: string;
  url: string
}
```

Reply:
対応いたしました！